src/components/List/List.jsx

```diff
@@ -10,13 +10,13 @@ import ListHeader from './ListHeader/ListHeader';
 
 const { iotPrefix } = settings;
 
-const itemPropTypes = {
+export const itemPropTypes = {
   id: PropTypes.string,
   content: PropTypes.shape({
     value: PropTypes.string,
     icon: PropTypes.node,
   }),
-  children: PropTypes.arrayOf(PropTypes.any), // TODO: make this recursive
+  children: PropTypes.arrayOf(PropTypes.any), // TODO: make this recursive on items?
```

Comment:
I don't think you can , so you can remove the comment

Reply:
I didn't think so. Changed it to simply `object`